src/views/menus/AccountMenu.tsx

```diff
@@ -59,12 +61,34 @@ export const AccountMenu = () => {
 
   const { evmAddress, walletType, dydxAddress, hdKey } = useAccounts();
 
+  const privy = usePrivy();
+
   const usdcBalance = freeCollateral?.current || 0;
 
   const onRecoverKeys = () => {
     dispatch(openDialog({ type: DialogTypes.Onboarding }));
   };
 
+  let walletIcon;
+  if (onboardingState === OnboardingState.WalletConnected) {
+    walletIcon = <Styled.WarningIcon iconName={IconName.Warning} />;
+  } else if (
+    onboardingState === OnboardingState.AccountConnected &&
+    walletType === WalletType.Privy
+  ) {
+    if (privy?.user?.google) {
+      walletIcon = <Icon iconComponent={GoogleIcon as ElementType} />;
+    } else if (privy?.user?.discord) {
+      walletIcon = <Icon iconComponent={DiscordIcon as ElementType} />;
+    } else if (privy?.user?.twitter) {
```

Comment:
nit: might be nicer to destructure the user object

Reply:
i looked into it but it was going to be harder with all the conditionals in here

Reply:
ah I meant

`const { google, discord, twitter } = privy?.user ?? {}`

then

`if (google), if (discord), if (twitter)`

lmk if that doesn't work

Reply:
that's clever, i'll change